src/Options/Options.php

```diff
@@ -203,7 +203,7 @@ protected function raw_value( $value ) {
 	 *
 	 * @return string
 	 */
-	protected function prefix_name( string $name ): string {
+	public function prefix_name( string $name ): string {
```

Comment:
Was there a specific intention to use this function outside the class? I'm not seeing it called anywhere, so we might as well leave it protected.

Edit: I saw it was mentioned in the additional section, but since it's not useful we might be better leaving it out.

Reply:
Thanks for catching it, it's a leftover from an alternative implementation using only `woocommerce_settings-google-for-woocommerce` callback, which I ultimately discarded.